### ISSUE TITLE
need some help understanding the loss curves

### ISSUE THREAD

**Issue by tino926** [meituan/YOLOv6#815]
### Before Asking

- [X] I have read the [README](https://github.com/meituan/YOLOv6/blob/main/README.md) carefully. 我已经仔细阅读了README上的操作指引。

- [X] I want to train my custom dataset, and I have read the [tutorials for training your custom data](https://github.com/meituan/YOLOv6/blob/main/docs/Train_custom_data.md) carefully and organize my dataset correctly; (FYI: We recommand you to apply the config files of xx_finetune.py.) 我想训练自定义数据集，我已经仔细阅读了训练自定义数据的教程，以及按照正确的目录结构存放数据集。（FYI: 我们推荐使用xx_finetune.py等配置文件训练自定义数据集。）

- [X] I have pulled the latest code of main branch to run again and the problem still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

### Search before asking

- [X] I have searched the YOLOv6 [issues](https://github.com/meituan/YOLOv6/issues) and found no similar questions.

### Question

I am currently training yolov6(3.0) on my own dataset. As shown in the following image, the training runs normally for the first 80 epochs, but after that, the training loss suddenly increases significantly at multiple points.

I am wondering what happened at these points. Can someone give me some possible explanations?

我目前正在使用自己的數據集訓練yolov6(3.0)。如下圖所示，在前80個epochs內訓練過程正常，但接下來即多次發生 training loss 暴增。

我不知該如何去了解在這些時間點發生了什麼，有人能給我一些可能的解釋嗎？或是可以如何改進我的訓練設定？

![image](https://user-images.githubusercontent.com/2639399/236780835-145e0256-034d-4d3a-b285-869846c4df00.png)

### Additional

I use this training command:
```bash
    python -m torch.distributed.launch --nproc_per_node 2 \
    tools/train.py --batch 64 --workers 16 \
    --conf configs/myconfig.py \
    --data data/mydata.yaml \
    --name myproj \
    --check-images --check-labels \
    --fuse_ab --device 0,1 --img 480 --epochs 600
```

with this model config: (same as yolov6n_finetune.py, except the depth_multiple)
```python
# YOLOv6s model
model = dict(
    type='YOLOv6n',
    pretrained='weights/yolov6n.pt',
    depth_multiple=0.2,
    width_multiple=0.25,
    backbone=dict(
        type='EfficientRep',
        num_repeats=[1, 6, 12, 18, 6],
        out_channels=[64, 128, 256, 512, 1024],
        fuse_P2=True,
        cspsppf=True,
        ),
    neck=dict(
        type='RepBiFPANNeck',
        num_repeats=[12, 12, 12, 12],
        out_channels=[256, 128, 128, 256, 256, 512],
        ),
    head=dict(
        type='EffiDeHead',
        in_channels=[128, 256, 512],
        num_layers=3,
        begin_indices=24,
        anchors=3, 
        anchors_init=[[10,13, 19,19, 33,23], 
                      [30,61, 59,59, 59,119], 
                      [116,90, 185,185, 373,326]],
        out_indices=[17, 20, 23],
        strides=[8, 16, 32],
        atss_warmup_epoch=0,
        iou_type='siou',
        use_dfl=False, # set to True if you want to further train with distillation 
        reg_max=0, # set to 16 if you want to further train with distillation
        distill_weight={
            'class': 1.0,
            'dfl': 1.0,
        },
    )
)

solver = dict(
    optim='SGD',
    lr_scheduler='Cosine',
    lr0=0.0032,
    lrf=0.12,
    momentum=0.843,
    weight_decay=0.00036,
    warmup_epochs=2.0,
    warmup_momentum=0.5,
    warmup_bias_lr=0.05
)

data_aug = dict(
    hsv_h=0.0138,
    hsv_s=0.664,
    hsv_v=0.464,
    degrees=0.373,
    translate=0.245,
    scale=0.898,
    shear=0.602,
    flipud=0.00856,
    fliplr=0.5,
    mosaic=1.0,
    mixup=0.243,
)
```

**Comment by shensheng272** [comment]
--epochs 600  导致100epoch左右的学习率还是很大，可能会出现波动。可以考虑减小学习率，或者减少epoch试试看。
通常的波动一波会下降3~5ap，直接降到0建议重新训练比较好。
